UserInterface.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// User interraction of the application.
+    /// </summary>
+    public class UserInterface
+    {
+        private const string notNumericMessage = "Please input numeric value.";
+        private const string pressAnyKeyMessage = "Press any key to continiue.";
+        private const string notInRangeMessage = "The value is not within range.";
+
+        /// <summary>
+        /// Gets numeric input from the user.
+        /// </summary>
+        /// <param name="prompt">Text to display for the user.</param>
+        /// <returns>Returns user input as int value.</returns>
+        public static int GetNumericInput(string prompt)
```

Comment:
methods should not be static, as there is no need for it here.

Reply:
Removed static from methods except Print method for easier use.

Reply:
please don't use static for easier use 😄 
in general for your projects the only static class that is allowed to be is a constant class with string values, all the other classes where should not be statis.

Reply:
Removed the static.